packages/services/api/src/modules/target/resolvers/Mutation/setTargetValidation.ts

```diff
@@ -22,9 +24,33 @@ export const setTargetValidation: NonNullable<MutationResolvers['setTargetValida
     enabled: input.enabled,
   });
 
-  return targetManager.getTarget({
+  const result = await targetManager.getTarget({
     organization,
     project,
     target,
   });
+
+  // Audit Log Event
+  const currentUser = await injector.get(AuthManager).getCurrentUser();
```

Comment:
you can re-write these into a `Promise.all` since these tasks can be done in parallel we don't need them to be sequential

```ts
const [result, currentUser] = await Promise.all([ targetManager.get(...), injector.get(...)])
```